docs/examples/browser-devenv-v3/files/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <link rel="stylesheet" href="style.css">
+  <title>Agregore self-hosted devenv V3</title>
+  <h1>Agregore self-hosted devenv V3</h1>
+  <button onclick="window.showEditor()">Show Editor</button>
+  <script src="lib.js"></script>
+  <script src="upload.js"></script>
+  <script src=" https://cdn.jsdelivr.net/npm/ace-builds@1.31.2/src-min-noconflict/ace.min.js "></script>
```

Comment:
It'd be good if we could bring the files into the source tree since this setup wouldn't load when offline. Or maybe have a fallback?

Reply:
I agree that would be nice. I decided to load the files from the CDN since adding them in the bootstrap would require adding about 550 files in the [manifest](https://github.com/AgregoreWeb/website/pull/54/files/a8915c4e48fb2194e4b211b76b62ba9bc506a47b#diff-aae0be0769b4b522189c7664bd4e7fa1d60b826a46b11bf6de24e413c8901b0aR15-R20) which wouldn't be nice. I really don't like the way ACE is distributed.

Reply:
Does this mean we cannot use this version of the editor while offline? Might be good to specify that if there's no easy alternative. The editor having a single `.min.js` from the CDN made me thing that getting a bundle would be possible.

Reply:
I've added the minimal set of files to support JavaScript, HTML, CSS and plain text without relying on the CDN!